kube/file-vault/file-vault-deployment.yml

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+  name: file-vault
+  {{ else if eq .KUBE_NAMESPACE .BRANCH_ENV}}
+  annotations:
+    downscaler/uptime: {{ .NON_PROD_AVAILABILITY }}
+  name: file-vault-{{ .DRONE_SOURCE_BRANCH }}
+  {{ else }}
+  annotations:
+    downscaler/uptime: {{ .NON_PROD_AVAILABILITY }}
```

Comment:
that would apply to UAT, STG and PROD environments, you would want to exclude at least PROD
 `downscaler/uptime: {{ .NON_PROD_AVAILABILITY }}`

Reply:
I will be adding this back in the next ticket the prod env should be ready by the end of week